### PR TITLE
GP2-599_header_menu

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [x] Ticket exists in Jira https://URL/browse/GP2-599 
 - [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added.

### CODE REVIEW

File: core/sass/components/_header.scss [hunk 1]
```diff
@@ -18,12 +18,8 @@
 }
 
 .great-mvp-header-logo {
```
Comment (dahfool) [inline core/sass/components/_header.scss:20]: `great-mvp-header` needs renaming.

**Review by lewis-coulson-dit — APPROVED** [review]
LGTM

File: core/sass/components/_product-finder.scss [hunk 2]
```diff
@@ -71,7 +71,7 @@
 /****** TEMPORARY Ghost/chevron buttons Will be replaced by great styles ******/
 .button {
   &--ghost-blue {
-    background: transparent;
+    background: white;
```
Comment (dahfool) [inline core/sass/components/_product-finder.scss:74]: `white` aka `$colour-white-100:`

Comment (bobmeredith) [reply core/sass/components/_product-finder.scss]: True - this is only a stopgap until new tag styles are available - but will change anyway.

File: core/sass/components/_header.scss (hunk 1 above)
Comment (bobmeredith) [reply core/sass/components/_header.scss]: Will do - although I guess this cleanup is a big job in total.
I'll rename the classes in the header.

**Pull request merged**